proxyclient/m1n1/proxyutils.py

```diff
@@ -86,17 +86,22 @@ def __init__(self, p, heap_size=1024 * 1024 * 1024):
         }
 
     def read(self, addr, width):
+        '''read(addr, width) - do a width read from addr and return it
+        width can be 8,16,21,64 or 132'''
```

Comment:
s/132/128/

Also please add spaces after commas for style.

Reply:
Ok - I note there are a few other, unrelated, instances found by : git grep -e '[^, ],[^, ]'

Reply:
Addressed in new commit